src/components/accession2/withdraw/WithdrawModal.tsx

```diff
@@ -464,49 +347,32 @@ export default function WithdrawDialog(props: WithdrawDialogProps): JSX.Element
             </Grid>
           </>
         ) : null}
-        <Grid item xs={12} paddingBottom={2}>
-          <Box display='flex' alignItems={fieldsErrors.withdrawnQuantity ? 'center' : 'end'}>
-            <Textfield
-              label={strings
-                .formatString(
-                  strings.AMOUNT_REMAINING,
-                  `${accession.remainingQuantity?.quantity} ${accession.remainingQuantity?.units}`
-                )
-                .toString()}
-              id='withdrawnQuantity'
-              onChange={(value) => onChangeWithdrawnQuantity(Number(value))}
-              type='text'
-              value={
-                isNurseryTransfer
-                  ? nurseryTransferRecord.germinatingQuantity
-                  : record.withdrawnQuantity?.quantity.toString()
-              }
-              errorText={fieldsErrors.withdrawnQuantity}
+        <Box className={classes.withdraw}>
+          <Grid item xs={12} textAlign='left'>
+            <Typography color={theme.palette.TwClrTxtSecondary} display='flex' fontSize={14}>
+              {strings.WITHDRAW_BY}
+            </Typography>
+            <RadioGroup
```

Comment:
Just a comment for future work, looks like a good candidate for web-components, there's some common layout across radio group usage.